linux/LinuxProcessList.c

```diff
@@ -704,10 +705,16 @@ static bool LinuxProcessList_readCmdlineFile(Process* process, const char* dirna
    int amtRead = xread(fd, command, sizeof(command) - 1);
    close(fd);
    int tokenEnd = 0;
+   int tokenStart = 0;
    int lastChar = 0;
    if (amtRead == 0) {
       if (process->state == 'Z') {
-         process->basenameOffset = 0;
+         if (!process->comm)
+            process->basenameOffset = 0;
+         else
```

Comment:
Please use braces for this if-statement as this is hard to read. Cf. discussion in #158.

Reply:
Ok.